lib/include/dlisio/lis/protocol.hpp

```diff
@@ -312,6 +312,18 @@ struct component_block {
 component_block read_component_block(const record&, std::size_t)
 noexcept (false);
 
+/* Information Records as defined in ch 3.3.1. Data Information Record and ch
+ * 4.1.3. Types 32, 34, 3: Information Records.
+ *
+ * dlisio's implementation of Information Records does *not* include Encrypted
+ * Table Dump- and Table Dump Records.
+ */
+struct information_record {
+    record_info info;
```

Comment:
In DFSR we need it to properly find the corresponding implicits when reading curves. In InfoRecs we keep the record type, which would otherwise be lost (or up to python to patch on again later). record_info is sufficiently small, and the total number of InfoRecs in a file is assumed to be relatively low, so the overhead is small enough to warrant the extra copy/memory Imo

Reply:
I dislike that from the abstraction point of view, even though it is useful.
But for now ok, it can stay. We can return to this design question in future, if we have, khem, resources.